website/blog/2025-01-20-podman-quadlet.md

```diff
@@ -0,0 +1,206 @@
+Fortunately, the Podman Desktop extension Podman Quadlet simplifies this process, enabling you to list, generate, and edit Quadlets visually.
+
+### Key Features of the Extension
+
+- **Integration with Podlet**: Generates Quadlets from existing Podman objects.
+- **Quadlet Management UI**: Provides a dedicated interface to list, edit, delete, start, and stop Quadlets.
+- **Logs Viewer**: Fetches and displays systemd logs using journalctl for troubleshooting.
+
+### Installation
+
+If you already have the latest version of Podman Desktop, you can <a href="podman-desktop:extension/podman-desktop.quadlet">**click here to install the Podman Quadlet extension**</a>.
+
+Alternatively, navigate to the Extensions page within Podman Desktop to install it
+
+### Listing Quadlets :clipboard:
+
+On the Podman Quadlet page, you can view all the Quadlets available across your Podman machines. To update the list, click **Refresh**.
+
+<ThemedImage
+alt="Quadlets List"
+sources={{
+    light: require('./img/podman-quadlet/podman-quadlet-home-light.png').default,
+    dark: require('./img/podman-quadlet/podman-quadlet-home-dark.png').default,
+  }}
+/>
+<br/><br/>
+
+In Podman Desktop, you can see the difference between containers managed by a Quadlet with a dedicated icon.
+
+<ThemedImage
+alt="Container Quadlet Icon"
+sources={{
+    light: require('./img/podman-quadlet/container-icon-quadlet-light.png').default,
+    dark: require('./img/podman-quadlet/container-icon-quadlet-dark.png').default,
+  }}
+/>
+
+### Generate Quadlets :hammer:
+
+To generate a Quadlet from an existing container, you’ll need to install [Podlet](https://github.com/containers/podlet). The extension simplifies this process:
+
+1. Go to **<Icon icon="fa-solid fa-cog" size="lg" /> Settings > CLI Tools** and install Podlet using the Podman Quadlet extension.
+
+<ThemedImage
+alt="Podlet Installation"
+sources={{
+    light: require('./img/podman-quadlet/cli-podlet-light.png').default,
+    dark: require('./img/podman-quadlet/cli-podlet-dark.png').default,
+  }}
+/>
+<br/><br/>
+
+Alternatively, you can download Podlet manually from its [GitHub release](https://github.com/containers/podlet/releases).
+The extension will detect the executable automatically.
+
+#### Example: Generating a Container Quadlet
+
+1. Start a container using Podman:
+
+```shell
+podman run --name nginx-demo -d -p 80:8080 nginx
+```
+
+2. In Podman Desktop, find your container on the Containers page.
+3. Use the action menu (**Actions > Generate Quadlet**) to create a Quadlet.
+
+<ThemedImage
+alt="Container actions"
+sources={{
+    light: require('./img/podman-quadlet/generate-quadlet-action-light.png').default,
+    dark: require('./img/podman-quadlet/generate-quadlet-action-dark.png').default,
+  }}
+/>
+<br/><br/>
+
+The generate form includes:
+
+- The container engine being used
+- The Quadlet type (e.g., `*.container`)
+- The resource (e.g., `nginx-demo`)
```

Comment:
Here, we need to continue the procedure rather than including these details which are already visible in the next screenshot. we can remove these details as it is breaking the procedural flow.

Reply:
I updated following your advise